backend/corpora/common/entities/dataset.py

```diff
@@ -148,28 +145,25 @@ def dataset_and_asset_deletion(self):
         self.tombstone_dataset_and_delete_child_objects()
 
     @staticmethod
-    def new_processing_status():
+    def new_processing_status() -> dict:
         return {
             "upload_status": UploadStatus.WAITING,
             "upload_progress": 0,
             "processing_status": ProcessingStatus.PENDING,
         }
 
-    def copy_csv_to_s3(self, csv_file):
-        s3 = boto3.resource("s3", endpoint_url=os.getenv("BOTO_ENDPOINT_URL"))
-        cellxgene_bucket = os.getenv("CELLXGENE_BUCKET", f"hosted-cellxgene-{os.environ['DEPLOYMENT_STAGE']}")
-        s3.meta.client.upload_file(csv_file, cellxgene_bucket, csv_file)
+    def copy_csv_to_s3(self, csv_file: str) -> str:
+        s3_file = f"{get_cxg_bucket_path(self.deployment_directories[0])}/genesets.csv"
+        cxg_bucket.upload_file(csv_file, s3_file)
```

Comment:
using new cxg_bucket

Reply:
Wont this still have the .cxg in the s3 file name? 
Also switching the suffix will require a corresponding change in cellxgene. Is there a reason for the change?

Reply:
it will still have cxg in the file name It would be `{dataset_id}.cxg/genesets.csv`. You want it to be  `{dataset_id}.cxg-genesets.csv`?

Reply:
{dataset_id}-genesets.csv
The previous code matched how cellxgene generates the name of the geneset file from dataset url